--- wf-config-0.8.0/meson.build.orig	2023-08-08 17:16:59.000000000 +0200
+++ wf-config-0.8.0/meson.build	2024-05-30 19:46:15.443026569 +0200
@@ -13,7 +13,11 @@
 
 add_project_arguments(['-Wno-deprecated-declarations'], language: ['cpp'])
 
-glm = dependency('glm')
+glm = dependency('glm', modules: ['glm::glm-header-only'], required: false)
+if not glm.found()
+    glm = dependency('glm')
+endif
+    
 evdev = dependency('libevdev')
 libxml2 = dependency('libxml-2.0')
 
